### DISCUSSION TITLE
(not bug) android builtin virus blocker: infected with evo-gen

### DISCUSSION THREAD

**Discussion by Unacceptium** [christianrowlands/android-network-survey#35] [General]
**Describe the bug**
new android has Avast antivirus built in.
that gold of softwares did indentified this app as a malware infected with evo-gen

**To Reproduce**
install app from f-droid on at least version 11 of android
thats it

**Expected behavior**
this IS expected because of what data the app logs...

**Screenshots**
![Screenshot_20240322_191703](https://github.com/christianrowlands/android-network-survey/assets/145642963/663f9ec7-a5fb-4f87-bd72-af55d5721d10)
![Screenshot_20240322_191709](https://github.com/christianrowlands/android-network-survey/assets/145642963/0c103145-9c3f-4fc2-a6e5-a9fffa115371)

**Smartphone (please complete the following information):**
 - Device: honor rky-lx1
 - Android Version: 12

**Additional context**
would be cool if the f-droid desc tells us that this can happen on install

**Comment by swanserquack** [comment]
I can also second this issue, with just the new update immediately being flagged by F-Secure as malware and being removed.

**Comment by christianrowlands** [comment]
Thank you for letting me know. 

You are correct @Unacceptium that access to SMS is necessary for some of the logging that is done. Just to be clear though, Network Survey does not send any text messages, nor does it install anything using as mentioned in the screenshot. In fact, you can deny all the SMS permissions if you are not using the CDR (call detail record) logging feature. The SMS permissions are only requested the first time you turn on CDR logging.


I can update the IzzyOnDroid description to include details about this alert happening. Is there anything else I should do to improve this?

**Comment by christianrowlands** [comment]
Actually, I am going to convert this issue to a discussion and link to it in the app description so we can keep the conversation going.
